CONTRIBUTING.md

```diff
@@ -141,3 +141,8 @@ To import Quarkus you need to define the JVM Max Heap Size (E.g. `-Xmx1g`)
 ```
 DevMojoIT require a few minutes to run but anything more than that is not expected. Make sure that nothing is running on 8080.
 
+* The native integration test for my extension didn't run in the CI
+
+If you create a new extension, it's normal as to parallelize the CI the native build has been split in multiple steps.
```

Comment:
```suggestion
Quarkus native builds have been split up in multiple steps to speed up the CI, because of this it is normal that you do not see the build for a new extension. 
```

Reply:
Even better I think: `In the interest of speeding up CI, the native build steps have been split into multiple steps. This means that each new extension needs to be configured explicitly in `azure-pipelines.yml` to have it's integration tests run in native mode`.

Reply:
I use @geoand sentence but I add the name of the stage to add more info in the FAQ

Reply:
Good call. It has better wording.